sway/sway_text_node.c

```diff
@@ -87,7 +87,7 @@ static void render_backing_buffer(struct text_buffer *buffer) {
 	}
 
 	float scale = buffer->scale;
-	int width = ceil(buffer->props.width * scale);
+	int width = ceil(get_text_width(&buffer->props) * scale);
```

Comment:
I think we need to update the `pixman_region32_union_rect()` call below to match the new size.

Reply:
Yeah good point, done.